packages/components/select/macro-options.json

```
@@ -49,6 +49,18 @@
       }
     ]
   },
+  {
+    "name": "value",
+    "type": "string",
+    "required": false,
+    "description": "Value for the option which should be selected. Use this as an alternative to setting the `selected` option on each individual item."
```

Could this maybe be a bit clearer? It's potentially a bit confusing, as both things are called `value`. 😬  Maybe we should spell out that this needs to exactly match the value of the option you want to be initially selected?

```suggestion
    "description": "Use this to make an option initially selected when the page loads. The value should exactly match the value of the option to be selected. This is an alternative to setting the `selected` boolean option on each individual item."
```